ci/benchmark/buildspec.yml

```diff
@@ -0,0 +1,17 @@
+# Build Spec for AWS CodeBuild CI
+
+version: 0.2
+
+phases:
+  install:
+    commands:
+      - apt-get update
+      - apt-get install sudo -y
+      - git clone https://www.github.com/pytorch/serve.git
+      - cd serve
+      - git checkout cw_metrics
```

Comment:
for the merged PR should we instead use the master branch?

Reply:
Thanks fixed